lady_luck.js

```diff
@@ -177,13 +177,6 @@ function nwodToText(outcome) {
 	return `Rolling ${outcome.pool}${notes.length > 0 ? ' (' + notes.join(", ") + ')' : ''}; ${suxxToWords(outcome.successes)}. _Individual results:_ ${newResults.join(', ')}`;
 }
 
-function padStart(str, len, val) {
-	while (str.length < len) {
-	 str = val + str;
-	}
-	return str;
-};
-
```

Comment:
Removing this looks like it makes sense (and was the initial point of your PR in our conversation)